src/runtime/components/elements/Link.vue

```diff
@@ -1,7 +1,14 @@
 <template>
-  <button v-if="!to" :type="type" :disabled="disabled" v-bind="$attrs" :class="inactiveClass">
+  <component
+    :is="as"
+    v-if="!to"
+    :type="type"
```

Comment:
Is the `type` prop here used for specifying the button type? If so, what about `v-bind`?

@benjamincanac

Reply:
Do you mean `v-bind="$attrs"`?

Reply:
Yes

Reply:
Maybe we should remove both `type` and `disabled` prop indeed, not sure to remember why I'd put them here in a first place. Will try to experiment!

Reply:
> Maybe we should remove both `type` and `disabled` prop indeed, not sure to remember why I'd put them here in a first place. Will try to experiment!

I remove `type` because it's unneeded; we can use `v-bind`, but `disabled` has logic in `Link`